### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by heather-everton**
Add Team and user deactivate routes

### CODE REVIEW

**Review by jesseparent — CHANGES_REQUESTED** [review]
Maybe get rid of Delete

File: controllers/api/team-routes.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            console.log(err);
+            res.status(500).json(err);
+        });
+});
+
+// UPDATE
+router.put('/:id', (req, res) => {
+    Team.update(
+        {
+            name: req.body.name,
```
Comment (jesseparent) [inline controllers/api/team-routes.js:43]: We should accommodate a project id

File: controllers/api/team-routes.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    )
+        .then(dbTeamData => res.json(dbTeamData))
+        .catch(err => {
+            console.log(err);
+            res.status(500).json(err);
+        });
+});
+
+// DELETE
+router.delete('/:id', (req, res) => {
```
Comment (jesseparent) [inline controllers/api/team-routes.js]: We do not want to support a Team delete for MVP

File: controllers/api/user-routes.js [hunk 3]
```diff
@@ -1,28 +1,30 @@
 const router = require('express').Router();
-const { User } = require('../../models');
+const { User, User_State, Role, Team } = require('../../models');
```
Comment (jesseparent) [inline controllers/api/user-routes.js]: You don't appear to be using these new Models

**Review by jesseparent — CHANGES_REQUESTED** [review]
Please review my comments

File: controllers/api/team-routes.js (hunk 2 above)
Comment (heather-everton) [reply controllers/api/team-routes.js]: Deleted this in team routes

File: controllers/api/user-routes.js (hunk 3 above)
Comment (heather-everton) [reply controllers/api/user-routes.js]: Removed the excess models from user routes

File: controllers/api/team-routes.js (hunk 1 above)
Comment (heather-everton) [reply controllers/api/team-routes.js:43]: I'm going to use project-routes to link the team to the project

**Review by jesseparent — APPROVED** [review]
Looks great - be sure to test!

**Pull request merged**